lte/gateway/c/connection_tracker/src/BUILD.bazel

```diff
@@ -0,0 +1,41 @@
+# Copyright 2021 The Magma Authors.
+
+# This source code is licensed under the BSD-style license found in the
+# LICENSE file in the root directory of this source tree.
+
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+cc_binary(
```

Comment:
where are all the test targets?

Reply:
They are all here!! (don't look too hard, there are no tests)